static/js/scroll_bar.js

```diff
@@ -1,9 +1,5 @@
 $(function () {
-    $("#stream-filters-container").perfectScrollbar({
-        suppressScrollX: true,
-        useKeyboard: false,
-        wheelSpeed: 0.5,
-    });
+    ui.set_up_scrollbar($("#stream-filters-container"));
```

Comment:
Wheelspeed is 0.5 here (ui default is 0.68). Not sure if significant.

Reply:
That's probably fine, but you should test.  I think this is the "manage streams" UI.

Reply:
I have manually tested and the difference was too small for me to notice, so I think it's fine.